deployment.md

```diff
@@ -77,7 +77,7 @@ server {
 
     error_page 404 /index.php;
 
-    location ~ ^/index\.php(/|$) { {
+    location ~ ^/index\.php(/|$) { 
```

Comment:
```suggestion
    location ~ ^/index\.php(/|$) {
```

Might as well remove the trailing whitespace while you're at it. I was about to create the same PR. :joy:

Also, just to follow the general PR structure, change the title to something along the lines of `[11.x] Remove duplicate curly brace in Nginx configuration example`